survey_calculator/models/survey_calculator_computation.py

```diff
@@ -0,0 +1,85 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Odoo, Open Source Management Solution
+#    This module copyright (C) 2015 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import api, models, fields
+
+
+class SurveyCalculatorComputation(models.Model):
+    _name = 'survey.calculator.computation'
+    _description = "Survey Calculator Computation"
+
+    @api.depends('survey_id')
+    def get_questions_available(self):
```

Comment:
Would this function work by using a related (survey_id.page_ids.question_ids) field?
Otherwise, would
``` self.question_ids = self.survey_id.page_ids.mapped(lambda p: p.question_ids) ``` 
cover this?